modules/nf-core/software/bwamem2/mem/main.nf

```diff
@@ -6,23 +7,20 @@ def options    = initOptions(params.options)
 process BWAMEM2_MEM {
     tag "$meta.id"
     label 'process_high'
-    label 'BWAMEM2_MEM'
     publishDir "${params.outdir}",
         mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
 
     conda (params.enable_conda ? "bioconda::bwa-mem2=2.1 bioconda::samtools=1.11" : null)
```

Comment:
I am guessing, the mulled container now has bwamem2.2? In that case: for conda also 2.2 is needed :)